microcosm_flask/errors.py

```diff
@@ -0,0 +1,111 @@
+"""
+Generalized error handling.
+
+"""
+from logging import getLogger
+
+from flask import jsonify
+from werkzeug.exceptions import default_exceptions
+
+
+error_logger = getLogger("errors")
```

Comment:
I'll probably switch the main Flask logger at some point, but I haven't tested that it's getting configured properly yet.